R/S3.R

```diff
@@ -950,6 +953,9 @@ pander.CrossTable <- function(x, caption = attr(x, 'caption'), digits = panderOp
     or <- (nrow(nt) - ts) / nr
     nt.nr <- nrow(nt)
     nc <- ncol(nt)
+    if (!total.r) {
```

Comment:
This might be `NULL` with `descr 1.1.1` and below, which case will throw an error here. We might want to add a check on that (IMO no need to check the version of `descr`, it's enough to check if this is `NULL` before the other comparisons).